src/metrics/listener.ts

```diff
@@ -223,7 +224,16 @@ export class MetricsListener {
     if (config.apiKeySecretARN !== "") {
       try {
         const { SecretsManager } = await import("@aws-sdk/client-secrets-manager");
-        const secretsManager = new SecretsManager();
+        const region = process.env.AWS_REGION;
+        let secretsParams: SecretsManagerClientConfig = {};
+        if (region) {
+          const isGovRegion = region.startsWith("us-gov-");
+          secretsParams = {
+            useFipsEndpoint: isGovRegion,
+            region,
+          };
+        }
+        const secretsManager = new SecretsManager(secretsParams);
```

Comment:
```suggestion
        const region = process.env.AWS_REGION;
        const secretsParams: SecretsManagerClientConfig = {
            region,
            useFipsEndpoint: region.startsWith("us-gov-");
        };
        const secretsManager = new SecretsManager(secretsParams);
```

Do you really need to pass the region here? Or doesn't it work it you don't pass it? Normally region is used by the AWS SDK by default from the env AFAIK, just trying to figure out if you really need it? Or is it bc since you're already parsing it?

`AWS_REGION` should always exists

Reply:
You're right, we don't need to pass region. 

tested manually on commercial+gov and works fine